packages/baseai/src/utils/memory/git-sync/handle-git-sync-memories.ts

```diff
@@ -58,7 +58,8 @@ export async function handleGitSyncMemories({
 
 	const allFilesWithContent = await loadMemoryFilesFromCustomDir({
 		memoryName,
-		memoryConfig: config
+		memoryConfig: config,
+		useGitRepo: true
```

Comment:
why do we need to pass this flag? `loadMemoryFilesFromCustomDir` is only used for git tracked memories.

Reply:
Yup. I have removed the flag as it is not needed.